### DISCUSSION TITLE
get provider

### DISCUSSION THREAD

**Discussion by ebennaga** [WalletConnect/walletconnect-monorepo#3610] [Developer Support] [answered]
how to get provider from v2 ? what is different between provider version 1 from web3.js with provider version 2 walletconnect ?

**Comment by glitch-txs** [comment, marked as answer]
v2 uses wagmi/viem by default instead of web3.js
https://wagmi.sh/

**Comment by ebennaga** [comment]
oke,thanks.. 

hey @glitch-txs please, answer my issues in https://github.com/orgs/WalletConnect/discussions/3612
